src/app/containers/FrontPageMain/index.jsx

```diff
@@ -48,17 +58,15 @@ const FrontPageMain = ({ frontPageData }) => {
         <VisuallyHiddenText id="content" tabIndex="-1" as="h1">
           {offScreenText}
         </VisuallyHiddenText>
-        <Grid>
-          <GridItemConstrainedLargeWithTopMargin>
-            {groups.map((group, index) => (
-              <FrontPageSection
-                key={group.title}
-                group={group}
-                sectionNumber={index}
-              />
-            ))}
-          </GridItemConstrainedLargeWithTopMargin>
-        </Grid>
+        <FrontPageWrapper>
```

Comment:
Here we will be using the [GridMaxWidthGroup4](https://github.com/bbc/simorgh/pull/4493/files#diff-a2df2ae2165edc2019fe63a16f9ce6fdR47) which will be defined in the Simorgh [Grid component](https://github.com/bbc/simorgh/pull/4493/files#diff-860eef0af93e58fa85364ccd46280087R32).